sdk/search/azure-search-documents/azure/search/documents/_utils.py

```diff
@@ -3,18 +3,24 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from azure.core.pipeline import policies
+from azure.core.pipeline.policies import BearerTokenCredentialPolicy, AsyncBearerTokenCredentialPolicy
 
-CREDENTIAL_SCOPES = ["https://search.azure.com/.default"]
+DEFAULT_AUDIENCE = "https://search.azure.com"
```

Comment:
can the audience `kwarg` be any custom value or a set of values? If it's the latter maybe we can have an enum defined for default, US Gov, China, etc.?

Reply:
users can pass custom values here